docs/modules/general/introduction.mdx

```diff
@@ -0,0 +1,29 @@
+---
+id: introduction
+title: Introduction
+---
+
+### History
+
+PGM originally started off in 2012 as a single game plugin specificaly built to run one map, Airship Battle.
+This plugin could manage the match and reset the map easily at the end of the game.
+Other maps were created later, but these all had to be hardcoded into the plugin.
+This was cumbersome and complicated, and there needed to be a new way to quickly program maps into the plugin using an easy to understand language and syntax.
+This problem would later meet its solution, the PvP Game Manager.
+Players unknown to Java programming could now create their own games and matches using an XML file that PGM takes as an input.
+As of now, there are around 1000+ maps made for PGM!
+
+### How it Works
+
+All maps made for PGM are programmed using an XML file called `map.xml`.
+XML is structured and layered silimar to HTML, making it easily readable.
+Everything from objectives, spawn kits, filters, jump pads, build areas, kill rewards and damage disable is defined in a map's XML file.
```

Comment:
seems like a weird list of modules to pick, maybe just teams, spawns, kits, and objectives as the basics

Reply:
I think most people would understand the concepts of a kill reward or jump pads, I just wanted to show that even more sophisticated stuff can be easily set.